### PR TITLE
[wip] Enable deleteList

### PR DESCRIPTION
**Pull request by TomHAnderson**
Pending unit tests. 

https://github.com/zfcampus/zf-rest/pull/59 and https://github.com/zfcampus/zf-content-negotiation/pull/29 need to be merged first.

### CODE REVIEW

**Comment by ezimuel** [comment]
@TomHAnderson I merged the PRs https://github.com/zfcampus/zf-rest/pull/59 and https://github.com/zfcampus/zf-content-negotiation/pull/29

**Comment by weierophinney** [comment]
This needs unit tests, please.

**Comment by weierophinney** [comment]
@TomHAnderson zfcampus/zf-rest#64 was merged earlier today, so that's no longer a bottleneck. Still need tests, though!

**Comment by TomHAnderson** [comment]
Requires https://github.com/zfcampus/zf-rest/pull/66 before merge

**Comment by weierophinney** [comment]
@TomHAnderson As noted on #66, it duplicates what was in #64, and that latter PR was merged against develop for v1.1 (as it contains BC breaks and bumps the required ZF version).

I'll pin this to `zfcampus/zf-rest:~1.1-dev` when merging and testing.

File: test/src/Server/ORM/CRUD/CRUDTest.php [hunk 1]
```diff
@@ -525,12 +525,73 @@ function (DoctrineResourceEvent $e) {
...
+            )
+        );
+        $this->getRequest()->setMethod(Request::METHOD_DELETE);
+        $this->getRequest()->setContent(json_encode($deleteList));
+        $this->dispatch('/test/artist');
+        $this->assertEquals(204, $this->getResponseStatusCode());
+
+        $this->validateTriggeredEventsContains(array(
+            DoctrineResourceEvent::EVENT_DELETE_PRE,
+            DoctrineResourceEvent::EVENT_DELETE_POST,
```
Comment (weierophinney) [inline test/src/Server/ORM/CRUD/CRUDTest.php]: The above two are not triggered by `deleteList()`, and are the source of the unit test failure. I've removed them locally for the merge review.

**Comment by weierophinney** [comment]
I've merged this to develop for release with 0.4.0 or 1.0.0, whichever we tag next; that way, current builds will continue to work against ZF 2.3.

**Pull request closed without merging**